config/metadata/configMeta.yaml

```diff
@@ -1211,7 +1213,7 @@ groups:
         v1name: MaxConnectionIdle
         type: duration
         valuetype: nondefault
-        default: 0s
```

Comment:
I did a global search in the codebase. It looks like we also need to update the one in `cofig_complete.yaml` for this value